qtconsole/call_tip_widget.py

```diff
@@ -5,6 +5,8 @@
 # System library imports
 from qtpy import QtCore, QtGui, QtWidgets
 
+from .util import IsQt6
```

Comment:
Please rename `IsQt6` to `IS_QT6`. That's because we don't use camel-case names for constants.